release/models/acl/openconfig-acl.yang

```diff
@@ -527,6 +550,15 @@ module openconfig-acl {
         "Description, or comment, for the ACL set";
     }
 
+    leaf counter {
```

Comment:
this leaf is not needed, as the grouping "access-list-entries-config" is used in both config and state containers.